### PR TITLE
Config file update (#12)

### PR DESCRIPTION
**Pull request by andrewmiroshnichenko**
### Changes related to commit #12 `Config file update`: 

1. User is able to control what pages to see, by extending `yeti.blockedPages` set. Possible values at this moment are `cdrs`, `accounts` and `rates`.
2. Router logic changed a bit, to allow more flexible route addition and filtering.
3. New store added to contain config-related things.

### Changes related to commit #13 `Vertical menu`:

1. Nav and main menu were split.
2. Nav menu turned into vertical shape.
3. Fancy styles applied to it.
4. Fixes for getters in store applied.
5. Pagination was hidden in case if there is only one page.
6. SASS support was added to templates.

Corresponding GIF:
navmenu

### Changes related to commit #14 `Hotfix for routing related things`:

1. Added fixes according to change requests https://github.com/yeti-switch/yeti-client/pull/15#discussion_r347131328 and https://github.com/yeti-switch/yeti-client/pull/15#discussion_r347130519.
2. Added link visibility check in newly created NavBar.
3. Constants for routing are used globally now.

### CODE REVIEW

File: src/router/helpers.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+});
+
+export const filterBlockedPages = () => ({
+  allowed: !store.getters.blockedPages.has('rates'),
+  routeInfo: '/',
+});
+
+
+export const beforeGuardEnchancer = (guardsArray) => (to, from, next) => {
+  const quardsResult = guardsArray.find((guard) => !guard(to, from).allowed);
```
Comment (segrey11) [inline src/router/helpers.js]: quards -> guards?

File: src/components/NavBar.vue [hunk 2]
```diff
@@ -106,6 +106,9 @@ export default {
     logout() {
       this.$store.dispatch('logout').then(() => this.$router.push('/login'));
     },
+    checkIfBreadcrumbIsDisabled(name) {
+      return !this.$store.getters.blockedPages.has(name);
```
Comment (segrey11) [inline src/components/NavBar.vue]: Judging by the name of the method it should return true if the page is blocked, but actually it does not. Would it be better to rename it to "isPageAllowed" or something similar?

File: src/router/helpers.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  },
+});
+
+export const requiresNotAuth = () => ({
+  allowed: !store.getters.isAuthenticated,
+  routeInfo: '/',
+});
+
+export const filterBlockedPages = () => ({
+  allowed: !store.getters.blockedPages.has('rates'),
```
Comment (segrey11) [inline src/router/helpers.js]: do we need here rates hardcoded?

Comment (andrewmiroshnichenko) [reply src/router/helpers.js]: No, thanks for catch

File: src/components/NavBar.vue (hunk 2 above)
Comment (andrewmiroshnichenko) [reply src/components/NavBar.vue]: Thanks, I used another name in last commit.

**Comment by andrewmiroshnichenko** [comment]
@segrey11 @dmitry-sinina fixes applied

**Pull request merged**